Svc/Ccsds/SpacePacketDeframer/SpacePacketDeframer.cpp

```diff
@@ -52,6 +52,9 @@ void SpacePacketDeframer ::dataIn_handler(FwIndexType portNum, Fw::Buffer& data,
     if (pkt_length > data.getSize() - SpacePacketHeader::SERIALIZED_SIZE) {
         FwSizeType maxDataAvailable = data.getSize() - SpacePacketHeader::SERIALIZED_SIZE;
         this->log_WARNING_HI_InvalidLength(pkt_length, maxDataAvailable);
+        if (this->isConnected_errorNotify_OutputPort(0)) {
+            this->errorNotify_out(0, Svc::Ccsds::FrameError::SP_INVALID_LENGTH);
```

Comment:
Could passing the `data, context` pair be useful as well? Maybe... but need to keep in mind that (1) it's likely corrupted and (2) it's about to be deallocated in the next line